infrastructure/stacks/development-pipeline/codebuild.tf

```diff
@@ -149,8 +149,8 @@ resource "aws_codebuild_project" "di_build" {
   }
 }
 
-resource "aws_codebuild_project" "di_deploy" {
-  name           = "${var.project_id}-${var.environment}-deploy-stage"
+resource "aws_codebuild_project" "di_deploy_dev" {
```

Comment:
The ticket says test environment and we are calling as dev here

Reply:
it is now and dev and test so i renamed the original to dev to distinguish it from test